Remove adb sdklib

cesarferreira opened the pull request.

cesarferreira committed:
removed adb sdklib

luispereira commented:
I will take advantage of this PR for refactoring and cleaning up the code regarding the removal of  adb lib

luispereira committed:
[Refactoring] - Removed  dryrun adb methods and merged with dryrun_utils

luispereira committed:
Fixed an issue where if no devices were selected, dryrun would install instead of assembling

luispereira committed:
Fixed an issue on windows which the adb could be busy. Added a mechanism to kill the adb and starting it again
Fixed an issue where all output were being taken into account (no need to listen for *  something * or everything that starts with adb)

luispereira commented:
I think its all for now, any issue you might have please let me know

cesarferreira commented on lib/dryrun/android_project.rb:
```diff
@@ -150,68 +157,68 @@ def sample_project
+    manifest_file = get_manifest(path_to_sample)
 
-      if manifest_file.nil?
-        return false
```
the `return` is optional in ruby, you could just wrote `false` or

``` ruby
return false if manifest_file.nil?
```

luispereira replied:
hey thanks for the tip, will use it on the future. thanks
